src/modules/components/member/memberDataListItem/memberDataListItemSkeleton/memberDataListItemSkeleton.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames';
+import { DataList, StateSkeletonBar, StateSkeletonCircular, type IDataListItemProps } from '../../../../../core';
+
+export interface IMemberDataListItemSkeletonProps extends IDataListItemProps {}
+
+export const MemberDataListItemSkeleton: React.FC<IMemberDataListItemSkeletonProps> = (props) => {
+    const { className, ...otherProps } = props;
+
+    return (
+        <DataList.Item
+            tabIndex={0}
+            aria-busy="true"
+            aria-label="loading"
+            className={classNames('flex flex-col gap-y-4 bg-neutral-0 py-3 md:py-3.5', className)}
```

Comment:
Why column with gap-y when you only have one child? Is the flexbox even necessary at this point?